chromium_src/third_party/blink/renderer/core/exported/web_document.cc

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "third_party/blink/public/web/web_document.h"
+#include "third_party/blink/renderer/core/dom/document.h"
+#include "third_party/blink/renderer/core/html/plugin_document.h"
+#include "third_party/blink/renderer/platform/bindings/script_state.h"
+
+#define IsPluginDocument                                                    \
```

Comment:
This doesn't need to be a #define, does it? 

Can just be added as a regular implementation, i.e:

<pre>
#include "src/third_party/blink/renderer/core/exported/web_document.cc"

namespace blink {

bool WebDocument::IsDOMFeaturePolicyEnabled(...) {
...
}

}  // namespace blink
</pre>